deploy_pulp3/pulp3_db/tasks/main.yml

```diff
@@ -0,0 +1,34 @@
+- name: Install postgres packages
+  package: name={{ item }} state=present
+  with_items:
+      - postgresql
+      - postgresql-server
+      # needed by postgres-related tasks and other things
+      # using python outside of a virtualenv
+      - python3-psycopg2
+
+- name: Set up postgres data dir
+  command: postgresql-setup --initdb
+  args:
+      creates: /var/lib/pgsql/data/base
+
+- name: Set up postgress access rules
+  copy:
+      src: pg_hba.conf
+      dest: /var/lib/pgsql/data/pg_hba.conf
+      owner: postgres
+      group: postgres
+      mode: 0600
+
+- name: Start and enable postgresql
+  service: name=postgresql state=restarted enabled=yes
+
+- name: Set up pulp DB user
+  postgresql_user:
+      name: pulp
```

Comment:
I think we should use the psql peer authentication method, which means this should match the unix user name.